src/components/Nav/Nav.js

```diff
@@ -1,10 +1,57 @@
+/* eslint-disable */
 import React, { Component } from 'react';
-import { withRouter } from 'react-router-dom';
+import './Nav.scss';
+import '../../styles/variables.scss';
+import { RiHome2Fill } from 'react-icons/ri';
+import { RiShoppingCartFill } from 'react-icons/ri';
+import { HiViewList } from 'react-icons/hi';
```

Comment:
1. 위 리뷰 참고하셔서 임포트 순서 맞춰주세요!
2. 아래와 같이 import 시 줄일 수 있는 부분이 있겠네요! 결국 객체임을 잊지 마세요!

```suggestion
import './Nav.scss';
import '../../styles/variables.scss';
import { RiHome2Fill, RiShoppingCartFill, HiViewList } from 'react-icons/ri';
```